tests/test_connection.py

```diff
@@ -213,6 +213,31 @@ def test_invalid_method_name(self):
         with self.assertRaises(SyncanoValueError):
             self.connection.make_request('INVALID', 'test')
 
+    @staticmethod
+    def _create_test_file():
```

Comment:
You could also use StringIO: https://docs.python.org/2/library/stringio.html
But I guess real or temp file creates a more "real" environment for tests even though StringIO has pretty much the same interface.